apps/newrelic.js

```diff
@@ -0,0 +1,36 @@
+var yaml = require('js-yaml');
+var fs = require('fs');
+var path = require('path');
+
+var licenseKey;
+try {
+  var locals = yaml.safeLoad(fs.readFileSync(path.join(__dirname, '..', 'locals.yml')));
+  licenseKey = locals.new_relic_license_key;
+} catch (e) {
+  // meh...
+}
+
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config.defaults.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
```

Comment:
This config only works from node.js. So it shouldn't have any impact on production newrelic since that runs in the browser.